File: Java_appli_codes/command.c
#include<stdio.h>
#include<stdlib.h>
//in c/cpp at 0th index(Arr[0]) contains name of executable where as in java directly starts with command argument
//argc - argument count
//the names argc & argv can be changed
int main( int argc, char *argv[])  //main is function which accepts two parameters first is interger and second is one dimentional array whose each element is character pointer
{
    int ans =0;
    printf("Name of executable is: %s \n",argv[0]);  //at address contains the name of executable

    printf("Number of command line arguments are: %d \n",argv);

     ans = atoi(argv[1]) + atoi(argv[2]);  //atoi- ascii to integer (contains in atoi header file)
    printf("Addition is: %d \n",ans);
    return 0;
}

//gcc command.c -o myexe
//myexe 11 21

File: Java_appli_codes/command.java
//Accept 2 no from user through command line display its addition
import java.lang.*;

public class command
 {
     public static void main(String Arg[]) 
    {
    System.out.println("NUmber of command line arguments are:"+Arg.length);
    System.out.println("First number is: "+Arg[0]);
    System.out.println("Second number is: "+Arg[1]); 
    
    int ans = Integer.parseInt(Arg[0]) + Integer.parseInt(Arg[1]);
    System.out.println("Addition is:"+ans);
    }
    
}
//javac command.java
//java command 10 11


